meshtastic/mesh_interface.py

```
@@ -3452,198 +3967,179 @@ def _handle_channel(self, channel: channel_pb2.Channel) -> None:
-        when a known protocol handler exists, decodes the protocol payload into
-        decoded.<handler.name> and attaches the protocol protobuf as
-        decoded.<handler.name>["raw"]. If a protocol handler defines an
-        onReceive callback it will be invoked with the packet dict. If the
-        decoded payload contains a requestId, any registered response handler
-        will be invoked (respecting ACK filtering and handler
-        ackPermitted/onAckNak rules). Finally, publishes the packet on a topic
-        like "meshtastic.receive", "meshtastic.receive.data.<portnum>", or
-        "meshtastic.receive.<protocol>" via the pub/sub system.
+        self,
+        meshPacket: mesh_pb2.MeshPacket,
+        hack: bool = False,
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)